src/components/Hero.js

```diff
@@ -2,24 +2,26 @@ import React, { useState } from "react";
 import styled from "styled-components";
 import { IoMdCloseCircle } from "react-icons/io";
 
-const close = (isVisible, setIsVisible) => {
-  setIsVisible(!isVisible);
-}
-
 export default function Hero(props) {
-  const [isVisible, setIsVisible] = useState(true)
+  const verifactHeroClosed = JSON.parse(localStorage.getItem('verifactHeroClosed')) ? true : false;
+  const [isClosed, setIsClosed] = useState(verifactHeroClosed)
   const welcomeTitle = "Your home for verifying credible news"
   const welcomeContent = "Post a question to our community of news sleuths to get answers and new perspectives about the news your reading"
 
-  return <>{isVisible ? (<Wrapper>
+  const close = () => {
+    localStorage.setItem('verifactHeroClosed', 'true');
+    setIsClosed(!isClosed);
```

Comment:
I would use an explicit `setIsClosed(true)` here. The method is called close, so it should only be able to go one way.

Reply:
Changed in `f980c26`